pages/phases/getting-started/measuring-a-hiring-action.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: article
+permalink: /hiring-phases/getting-started/collecting-data/
+section: hiring-phases
+title: Collecting data
+sidenav: hiring-phases
+phase: getting-started
+sub-phase-order: 030
+---
+
+## Collecting baseline data
+
+<p class="usa-intro">
+  To gauge the success of your hiring action, collect baseline data about recent delegated examination (DE) hiring actions at your agency. You'll compare this information with the results you receive when your new hiring action is complete.
+</p>
+
+For one to two recent DE actions, gather:
+
+1. **Current qualified rate**: How many applications were considered qualified by an HR specialist.
+2. **Current selection rate**: The number of selections made off of the resulting certificate, represented numerically and as a percentage of qualified applications.
```

Comment:
also, time to hire

Reply:
They already measure time to hire. In fact, that may be the only thing they do measure. Do we really need to add it?